ios/NotifyingMaskedTextFieldListener.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  NotifyingMaskedTextFieldListener.swift
+//  react-native-advanced-input-mask
+//
+//  Created by Ivan Ignathuk on 16/10/2024.
+//
+
+import ForkInputMask
+import Foundation
+import UIKit
+
+class NotifyingMaskedTextFieldListener: MaskedTextInputListener {
+  override func textField(
+    _ textField: UITextField,
+    shouldChangeCharactersIn range: NSRange,
+    replacementString string: String
+  ) -> Bool {
+    defer {
+      NotificationCenter.default.post(name: UITextField.textDidChangeNotification, object: textField)
```

Comment:
Are you sure it's needed?

Starting from 1.12+ `keyboard-controller` injects own delegate that intercepts all events, including `onChangeText` event 🙃

Reply:
Hmm, are you sure you need to insert this in your library, rather than it should be controlled from a box of text inputs? 🙂

Reply:
> Hmm, are you sure you need to insert this in your library, rather than it should be controlled from a box of text inputs? 🙂

Well, actually it's up to you (as a dev of this library)

MaskedTextFieldDelegate definitely break event propagation, so theoretically this fix is needed if other libs relies on this event

But `keyboard-controller` already uses a different approach to intercept text-change events (it inject own delegate, lol), so this fix is not required for `keyboard-controller` to work properly

Reply:
Let's merge this anyway, as masked input breaks notifications by design - I think other libs (such as inactivity if it would be written on native side) may depend on that, so it would be better to have it 👀